### DISCUSSION TITLE
How to write document graphql api and display it in "graphiql"?

### DISCUSSION THREAD

**Discussion by blatoo** [apollographql/apollo-server#4670] [Q&A]
Hi all, 

I am very new in graphql world. How can I document my graphql apis, so that they will be displayed in "graphiql" ?

Thanks very much for the help!

Conny

**Comment by delong8** [comment]
GraphQL spec has describe the standard way to document schema, https://spec.graphql.org/June2018/#sec-Descriptions 

try this:

```graphql

"""
complex description
"""
type NewType {
    " simple description "
    name: String
}
```
